src/audio-worklet/free-queue/src/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "freequeue",
+  "version": "1.0.0",
+  "license": "MIT",
+  "source": "src/index.ts",
+  "main": "dist/freequeue.cjs",
```

Comment:
I don't see some of files here. Can you check?

Reply:
because of .gitignore in source folder, typescript compiled files and build folder is ignored